test/e2e/azure/account.go

```diff
@@ -0,0 +1,124 @@
+package azure
+
+import (
+	"encoding/json"
+	"log"
+	"os/exec"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Account holds the values needed to talk to the Azure API
+type Account struct {
```

Comment:
Okay, it just sort of duplicates the auth stuff that `acs-engine deploy` now does directly and is a fair amount of code that could just be skipped if we called `acs-engine deploy` directly.